guide/react-native/installation.mdx

```diff
@@ -7,55 +7,87 @@ import TabItem from "@theme/TabItem";
 
 # Installation
 
-If you don't have an existing project, you can create a new Expo app using a template:
+## Create new App
 
-<Tabs groupId="package-managers">
-  <TabItem value="npm" label="NPM">
-  
-    ```
-    npx create-expo-app@latest my-video-app
-    ```
+<details>
+  <summary>Follow these steps to create new mobile app</summary>
 
-  </TabItem>
-  <TabItem value="yarn" label="YARN">
+If you don't have an existing project, you can create a new Expo app using a template
 
-    ```
-    yarn create expo-app my-video-app
-    ```
+```bash
+npx create-expo-app@latest my-video-app
+```
 
-  </TabItem>
-</Tabs>
+As next step, you have to generate native files with `expo prepare` command:
+
+```bash
+npx expo prebuild
+```
 
-## Installation
+You can also follow more detailed [expo instructions](https://docs.expo.dev/get-started/introduction/).
 
-### Step 1: Install the package
+</details>
+
+## Step 1: Install the package
 
 Install `@fishjam-cloud/react-native-client` package:
 
-<Tabs groupId="package-managers">
+```bash npm2yarn
+npm install @fishjam-cloud/react-native-client
+```
 
-  <TabItem value="npm" label="NPM">
+## Step 2: Configure App permissions
 
-    ```
-    npm install @fishjam-cloud/react-native-client
-    ```
+Your app will need to have configured permissions to use microphone and camera in order to work.
 
-  </TabItem>
-  <TabItem value="yarn" label="YARN">
+### Android
+
+For android you need to ask for certain permissions:
 
-    ```
-    yarn add @fishjam-cloud/react-native-client
-    ```
+- `android.permission.CAMERA`
+- `android.permission.RECORD_AUDIO`
+- `android.permission.MODIFY_AUDIO_SETTINGS`
+
+<Tabs groupId="app-type">
+
+  <TabItem value="expo" label="Expo">
+
+You need to modify `app.json` file and add all permissions:
+
+```json
+{
+  "expo": {
+    ...
+    "android": {
+      ...
+      "permissions": {
+        "android.permission.CAMERA",
+        "android.permission.RECORD_AUDIO",
+        "android.permission.MODIFY_AUDIO_SETTINGS"
+      }
+    }
+  }
+}
+```
 
   </TabItem>
-</Tabs>
+  <TabItem value="rn" label="Bare workflow">
 
-### Step 2: Configure screen sharing
+You need to modify AndroidManifest.xml file, and these lines:
 
-If you plan to use Screen Sharing feature, there are few extra steps required to configure it.
+```xml
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+  ...
+  <uses-permission android:name="android.permission.CAMERA"/>
+  <uses-permission android:name="android.permission.RECORD_AUDIO"/>
+  <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTING"/>
+  ...
+</manifest>
+```
 
-:::warning missing docs
+  </TabItem>
+</Tabs>
 
-This part of docs will be updated
+### iOS
 
-:::
+No action is required for iOS
```

Comment:
**issue:** On iOS you need `NSCameraUsageDescription` and `NSMicrophoneUsageDescription` in `Info.plist`, otherwise the app will crash.